addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -987,15 +988,15 @@ function makeActionManager(env) {
     }
 
     /**
-     * Helper function to find the first mobile-friendly view, if any.
+     * Helper function to find the mobile view, if any.
      *
      * @private
      * @param {Array} views an array of views
      * @param {boolean} multiRecord true if we search for a multiRecord view
      * @returns {Object|undefined} first mobile-friendly view found
      */
-    function _findMobileView(views, multiRecord) {
-        return views.find((view) => view.isMobileFriendly && view.multiRecord === multiRecord);
+    function _findMobileView(views, multiRecord, default_mobile_mode) {
```

Comment:
`_findView(views, multiRecord, viewType)`
also, don't forget to adapt the doctstring